### DISCUSSION TITLE
Move code to GitLab or Codeberg

### DISCUSSION THREAD

**Discussion by julianfairfax** [InfiniTimeOrg/InfiniTime#1947] [Ideas]
It would be nice if the project's code were moved to GitLab or Codeberg. GitHub is not an open source platform, and being owned by Microsoft makes it more of a danger to open source projects, rather than a help.

**Comment by JF002** [comment]
We thought about this when Microsoft bought Github : should we move to another platform? 

The thing is that moving the code to another platform will be straightforward since this is just a git repo. However, moving the CI, the pull-requests, the issues, discussions, tools and community will take time and energy. And this wouldn't bring any additional benefits.
So we decided to stay on Github, and we will reconsider this decision if Github/Microsoft would become hostile to the project or to the FOSS community or if it doesn't meet our needs anymore.
